devel/mrs/Makefile

```
@@ -0,0 +1,61 @@
+PORTNAME=	mrs
+DISTVERSION=	20221216
+CATEGORIES=	devel
+PKGNAMESUFFIX=
+FLAVORS=	cherivoke cherivoke_stats \
+		cornucopia cornucopia_stats \
+		cornucopia_load cornucopia_load_stats
+FLAVOR?=	cornucopia_load
+
+PKGNAMESUFFIX=	-${FLAVOR}
+
+MAINTAINER=	nfilardo@microsoft.com
+COMMENT=	Brett Gutstein's CHERI Temporal Safety Malloc Revocation Shim
+WWW=		https://github.com/ctsrd-cheri/mrs
+
+LICENSE=	BSD2CLAUSE MIT
+LICENSE_COMB=	multi
+
+USES=		cmake compiler:c++17-lang
+
+PLIST_SUB=	PKGNAMESUFFIX=${PKGNAMESUFFIX}
+
+USE_GITHUB=	yes
+GH_ACCOUNT=	nwf-msr
+GH_PROJECT=	mrs
+GH_TAGNAME=	d77220664c077d0568019eeb5f4d1d1dacea5343
+
+ALL_TARGET=	mrs
+
+CMAKE_ARGS+=	-DGIT_SHA1_OVERRIDE=${GH_TAGNAME}
+
+.if ${FLAVOR:M*_ratio*} == ""
```

```suggestion
.if !${FLAVOR:M*_ratio*}
```
I tend to prefer using :M as a boolean operator in these kinds of things.